README.md

```diff
@@ -69,18 +69,28 @@ make dist
 
 ``` shell
 sudo apt-get install build-essential libsdl2-dev libglew-dev
-cd platforms/linux
+cd platforms/unix
```

Comment:
I rather prefer an additional "bsd" directory in "platforms" instead of modifying the "linux" one as it will confuse some user and break some CI processes. 

https://github.com/drhelius/Gearboy/pull/140/checks?check_run_id=2982713141

Reply:
OK. I feel weird having two directories that contain the exact same files, but not breaking CI makes sense.